src/store/flow/flow.ts

```diff
@@ -16,20 +17,63 @@ import moment from 'moment'
 import { ActionTree, GetterTree, MutationTree } from 'vuex'
 import { IRootState } from '@/store'
 import {
-  defaults, includes, forEach, cloneDeep, get, has,
+  defaults, 
+  includes, 
+  forEach, 
+  cloneDeep, 
+  get, 
+  has, 
+  omit
 } from 'lodash'
 import { discoverContentTypesFor } from '@/store/flow/resource'
 import { computeBlockPositionsFrom } from '@/store/builder'
 import { IFlowsState } from '.'
 
 export const getters: GetterTree<IFlowsState, IRootState> = {
-  activeFlow: (state) => state.flows.length && getActiveFlowFrom(state as unknown as IContext),
-
+  //We allow for an attempt to get a flow which doesn't yet exist in the state - e.g. the firstFlowId doesn't correspond to a flow
+  activeFlow: (state) => {
+    if(state.flows.length) { 
+      try {
+        return getActiveFlowFrom(state as unknown as IContext)
+      } catch(err) {
+        return null
+      }
+    }
+  },
+  //TODO - is the IContext equivalent to the Flow Container? Can we say that it should be?
+  activeFlowContainer: state => {
+    return {
+      isCreated: state.isCreated,
+      specification_version: "TODO",
+      uuid: "TODO",
+      name: "TODO",
+      description: "TODO",
+      platform_metadata: {},
+      flows: state.flows,
+      resources: state.resources
+    } as unknown as IContext
```

Comment:
This will become IContainer when the flow-runner version bump is merged. These TODO fields all need to be deleted for now based on discussion in backend runtime hilda. CC @bzabos . I'll add in the latest spec version now it's settled a bit. Is there anywhere good we can pull this from dynamically?

Reply:
- Will do this in when moving to IContainer after this is merged to master and the flow-runner version bump is also merged there.